Inc/ST-LIB_LOW/ErrorHandler/ErrorHandler.hpp

```diff
@@ -0,0 +1,55 @@
+/*
+ * ErrorHandler.hpp
+ *
+ *  Created on: Dec 22, 2022
+ *      Author: Pablo
+ */
+
+#pragma once
+
+#include "C++Utilities/CppUtils.hpp"
+#include "Time/Time.hpp"
+#include "Communication/UART/UART.hpp"
+
+
```

Comment:
```suggestion

using std::snprintf;
using std::make_unique;
using std::string;

```
Mas limpio poner en el hpp lo que vas a gastar y no poner std:: en las implementaciones, pienso que deberia ser el convenio para la libreria

Reply:
Listo